src/components/users/ResetPasswordForm.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Loader from 'react-loader-spinner';
+import { object, string } from 'prop-types';
+
+import { userActions } from '../../actions/user.actions';
+import '../../style/App.scss';
+import './user-form.scss';
+import FormInput from '../common/FormInput';
+import { userRequest } from '../../actions/user.actions';
+
+const ResetPasswordForm = ({ intl, url_uid, url_token }) => {
+  const dispatch = useDispatch();
+  const [inputs, setInputs] = useState({
+    new_password1: '',
+    new_password2: '',
+  });
+  const { new_password1, new_password2 } = inputs;
+
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  const equalPasswords = new_password1 === new_password2;
+
+  const { loading, requestError, errorMsg, updated } = useSelector(
+    (state) => state.user
+  );
+
+  useEffect(() => {
+    if (updated) {
+      setIsSubmitted(false);
+      alert(
+        intl.formatMessage({
+          id: 'reset.password.completed.text',
+        })
```

Comment:
I think you have to redirect to the login page with a successful message in there

Reply:
I do this in the action. It's ok?

Reply:
I prefer to put the  logic on the components and leave actions to only manage async flow data but it's just a preference